src/api/generate-xlsx/map-application-to-XLSX/map-exporter/index.ts

```diff
@@ -79,6 +79,12 @@ export const mapBroker = (application: Application) => {
  */
 const mapExporter = (application: Application) => {
   const { company, companySicCodes, business } = application;
+  let financialYearEndDate = '';
+
+  // financial year end date should only be populated if it exists
+  if (company[FINANCIAL_YEAR_END_DATE]) {
```

Comment:
It looks like we may need to adjust `XLSX_ROW_INDEXES` from this change; some rows in the XLSX should be bold with a larger heading. It has to be done by row indexes, so if some rows become conditional we need to test some scenarios and adjust accordingly

Reply:
Discussed - the field will always exist now, so need for this 🎉